src/java/map/FloatMask.java

```diff
@@ -176,6 +176,24 @@ public FloatMask copy() {
         }
     }
 
+    public FloatMask clear() {
+        for (int x = 0; x < getSize(); x++) {
+            for (int y = 0; y < getSize(); y++) {
+                set(x, y, 0);
+            }
+        }
+        applySymmetry();
+        VisualDebugger.visualizeMask(this);
+        return this;
+    }
+
+    public FloatMask replaceWith(FloatMask other) {
+        clear();
+        add(other);
+        VisualDebugger.visualizeMask(this);
+        return this;
+    }
```

Comment:
This is inefficient as it is looping through twice where only a single loop through is needed. Additionally this functionality already exists if you do maskName = new FloatMask(other, seed);